daemon/daemon_windows.go

```
@@ -0,0 +1,114 @@
+package daemon
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/daemon/graphdriver"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/docker/docker/runconfig"
+	"github.com/docker/docker/volumes"
+)
+
+const runDir = os.Getenv("TEMP")
+
+func (daemon *Daemon) Changes(container *Container) ([]archive.Change, error) {
+	return daemon.driver.Changes(container.ID, container.ImageID)
+}
+
+func (daemon *Daemon) Diff(container *Container) (archive.Archive, error) {
+	return daemon.driver.Diff(container.ID, container.ImageID)
+}
+
+func parseSecurityOpt(container *Container, config *runconfig.HostConfig) error {
+	return nil
+}
+
+func (daemon *Daemon) createRootfs(container *Container) error {
+	// Step 1: create the container directory.
+	// This doubles as a barrier to avoid race conditions.
+	if err := os.Mkdir(container.root, 0700); err != nil {
+		return err
+	}
+	if err := daemon.driver.Create(container.ID, container.ImageID); err != nil {
+		return err
+	}
+	return nil
+}
+
+func checkKernel() error {
+	return nil
+}
+
+func (daemon *Daemon) verifyHostConfig(hostConfig *runconfig.HostConfig) ([]string, error) {
+	// TODO Windows. Verifications TBC
+	return nil, nil
+}
+
+// checkConfigOptions checks for mutually incompatible config options
+func checkConfigOptions(config *Config) error {
+	if config.Bridge.Iface != "" && config.Bridge.IP != "" {
+		return nil, fmt.Errorf("You specified -b & --bip, mutually exclusive options. Please specify only one.")
```

This was a cut and paste error. It should return an error only. Corrected